numpy/_core/_methods.py

```diff
@@ -98,8 +98,8 @@ def _count_reduce_items(arr, axis, keepdims=False, where=True):
 
 def _clip(a, min=None, max=None, out=None, **kwargs):
     if min is None and max is None:
-        raise ValueError("One of max or min must be given")
-
+        # TODO: is there a better way to return identity?
```

Comment:
This seems fine.

But it implies a (small) API change, which perhaps is worth a changelog entry?

Reply:
But then `out` argument is ignored. But what if a user passed it? Throw error?

I wonder about something like:
```python
def identity(x, out=None):
    if out is None:
        return x
    else:
        # TODO: move x contents to out
        ...
        return out
```

Reply:
Ah, yes, you are right. And perhaps in general one would expect a copy, like for `np.minimum` and `np.maximum`. In that case, how about `return np.positive(x, out=out)`?

Reply:
`np.positive` will work here - thank you!